### ISSUE TITLE
Bug with receiver

### ISSUE THREAD

**Issue by daimonkor** [whilu/LMBluetoothSdk#3]
``` java
    @Override
    protected void onCreate(){
              mBLEController = BluetoothLEController.getInstance().build(this);
              mBLEController.setBluetoothListener(bluetoothLEListener);
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();

        try {
            mBLEController.release();
        }catch (Exception e){
            LogManager.LogEA(e);
        }
    }

     @Override
    protected void onPause() {
        super.onPause();
       /*my custom key, use when scanLe*/
        if(getCurrentState() == START_SCAN_STATE) {
            mBLEController.cancelScan();
        }
    }
```

![image](http://i.imgur.com/byAyeOK.png)

Please fix, thank you for support.
I am using [BLE Simulator](https://play.google.com/store/apps/details?id=io.github.webbluetoothcg.bletestperipheral&hl=ru) (Battery Service).

**Comment by daimonkor** [comment]
Please look up (my example code), sometimes my listener did not register
You can see well scan, but not call your callback function
![image](http://i.imgur.com/2eq8bxN.png)
Please fix, thanks.

**Comment by whilu** [comment]
Thanks! I will fix the bug as soon as possible.

**Comment by andrew-boyarshin** [comment]
Encountered the bug too. Easy to reproduce by rotating the screen.

**Comment by daimonkor** [comment]
Hi, have you news about bug?

**Comment by whilu** [comment]
Sorry, late to reply. @STALKER2010 When u rotate the screen, have u released the instance? @daimonkor I can't reappear the bug u mentioned, can u show me more detail? Now I have removed the `registerReceiver()` method in BluetoothLEController, it's no use. If u have any problem, u can mail to me, I will reply you soon! Thank very much!

**Comment by daimonkor** [comment]
@whilu Did you create new release for gradle version?

**Comment by whilu** [comment]
@daimonkor If there is no any other issues, I will create a new release soon.

**Comment by andrew-boyarshin** [comment]
@whilu yes, I release instance. I tried it many ways, now I just implemented most Activity lifecycle callbacks (onDestroy, onPause etc) and I call super implementation in `try {...} catch (e) {...}` block. It doesn't crash, but it certainly doesn't seem to be working correctly.
